src/test/java/com/rtb/projectmanagementtool/task/TaskControllerTest.java

```diff
@@ -61,6 +67,58 @@ public void tearDown() {
     helper.tearDown();
   }
 
+  @Test
+  public void testGetTaskByID() {
+    DatastoreService ds = DatastoreServiceFactory.getDatastoreService();
+
+    // Assert no task entities are found
+    Assert.assertEquals(0, ds.prepare(new Query("Task")).countEntities(withLimit(10)));
+
+    // Add task entities to ds
+    ds.put(task1.toEntity());
+    ds.put(task2.toEntity());
+    ds.put(task3.toEntity());
+
+    // Assert 3 entities were added
+    Assert.assertEquals(3, ds.prepare(new Query("Task")).countEntities(withLimit(10)));
+
+    // Get task with TaskController
+    TaskController taskController = new TaskController(ds);
+    TaskData getTask = taskController.getTaskByID(taskID2);
+
+    // Assert task retrieved is correct
+    Assert.assertEquals("getTask", task2, getTask);
+  }
+
+  @Test
+  public void testGetSubtasks() {
+    DatastoreService ds = DatastoreServiceFactory.getDatastoreService();
+
+    // Subtasks of task1
+    ArrayList<Long> subtasksParameter = new ArrayList<>(Arrays.asList(2l, 3l));
```

Comment:
nit:  we don't need to tag as a "parameter".  If this was used later on for a non-param usages, it won't be accurate anymore.